staging/src/k8s.io/dynamic-resource-allocation/kubeletplugin/draplugin.go

```diff
@@ -335,6 +340,33 @@ func Start(ctx context.Context, nodeServers []interface{}, opts ...Option) (resu
 		}
 	}()
 
+	for _, nodeServer := range nodeServers {
+		if _, ok := nodeServer.(DRADriverWithCheck); !ok {
```

Comment:
Let's do this without runtime type assertions. Unmodified DRA drivers should fail to build at compile time.